### ISSUE TITLE
Button Type

### ISSUE THREAD

**Issue by DevKingDigital** [JosephusPaye/Keen-UI#43]
Is there a way to set the type on a button? It default to submit but I want to explicitly declare it as a button

**Comment by JosephusPaye** [comment]
At the moment, there's no prop for directly setting the button type. A `button-type` prop will be added in the next release.

Meanwhile, you can set the type using Javascript:

HTML
```html
<ui-button v-el:add-post-button>Add Post</ui-button>
```

JS: in Vue component `ready` method:

```js
ready: function() {
    this.$els.addPostButton.setAttribute('type', 'button');
}
```

**Comment by JosephusPaye** [comment]
New `buttonType` prop now available in [v0.8.9](https://github.com/JosephusPaye/Keen-UI/releases/tag/v0.8.9), see the [UiButton docs](https://josephuspaye.github.io/Keen-UI/#/ui-button-docs).
